### PR TITLE
Bump version to 5.11

### PR DESCRIPTION
**Pull request by finagolfin**
Match the compiler, apple/swift#68583.

### CODE REVIEW

**Comment by neonichu** [comment]
We won't be able to merge this for a little bit since we are auto-merging main to 5.10 for the moment, contrary to the compiler.

**Comment by finagolfin** [comment]
Ping, a CI run and this can go in now?

**Comment by MaxDesiatov** [comment]
Closing as superseded by https://github.com/apple/swift-package-manager/pull/7184#issuecomment-1852713829

File: Sources/PackageModel/ToolsVersion.swift [hunk 1]
```diff
@@ -31,6 +31,7 @@ public struct ToolsVersion: Equatable, Hashable, Codable, Sendable {
     public static let v5_8 = ToolsVersion(version: "5.8.0")
     public static let v5_9 = ToolsVersion(version: "5.9.0")
     public static let v5_10 = ToolsVersion(version: "5.10.0")
+    public static let v5_11 = ToolsVersion(version: "5.11.0")
```
Comment (finagolfin) [inline Sources/PackageModel/ToolsVersion.swift:34]: Note that #7184 is missing this change.

Comment (MaxDesiatov) [reply Sources/PackageModel/ToolsVersion.swift:34]: cc @neonichu

**Pull request closed without merging**